elide-contrib/elide-dynamic-config-helpers/src/main/resources/elideDBConfigSchema.json

```diff
@@ -70,9 +70,9 @@
                         "type": "string",
                         "title": "Elide Dialect",
                         "description": "The Elide Dialect to use for query generation.",
-                        "pattern": "^([0-9A-Za-z_]+[.]?)+$",
+                        "pattern": "^([h|H]2|[h|H][i|I][v|V][e|E]|[p|P][r|R][e|E][s|S][t|T][o|O])$",
```

Comment:
I think ENUM would be better in the long run.

Reply:
changed to ENUM, will allow upper case only